docs-src/concepts/what-is-a-schedule.md

```diff
@@ -8,8 +8,14 @@ tags:
   - explanation
 ssdi:
   - Introduced in Temporal Server version 1.17.0
-  - Available in tctl v1.17 and Temporal CLI
-  - Available in Temporal Cloud via tctl and CLI
+  - Available in Temporal CLI (and tctl v1.17)
+  - Available in Temporal Cloud
+  - Available in [Go SDK](/dev-guide/go/features#schedule-a-workflow)
+  - Available in [Python SDK](/dev-guide/python/features#schedule-a-workflow)
+  - Available in [TypeScript SDK](https://github.com/temporalio/samples-typescript/tree/main/schedules#schedules)
+  - Available in [.NET SDK](https://dotnet.temporal.io/api/Temporalio.Client.Schedules.html)
+  - Available in [gRPC API](https://api-docs.temporal.io/#temporal.api.workflowservice.v1.CreateScheduleRequest)
```

Comment:
Single sentence list?
```
- Available in the [Go SDK](/dev-guide/go/features#schedule-a-workflow), [Python SDK](....
```

Reply:
I like the list readability

https://temporal-documentation-prydek87n.preview.thundergun.io/workflows#schedule

<img width="604" alt="image" src="https://github.com/temporalio/documentation/assets/251288/753b448e-a0f2-4761-b517-c6c5df638ec0">